.github/workflows/maven.yml

```diff
@@ -22,10 +22,7 @@ jobs:
           distribution: ${{ matrix.dist }}
           cache: 'maven'
 
-      - name: Maven Package
+      - name: Maven Verify
         env:
           MAVEN_OPTS: -Dmaven.artifact.threads=64
-        run: mvn -V package --no-transfer-progress
-
-      #- name: Maven javadoc
-      #  run: mvn javadoc:javadoc -Ddoclint=all --no-transfer-progress
+        run: mvn -V verify --no-transfer-progress
```

Comment:
we don't need verify AFAIK so let's not misguide users/us with an irrelevant command in the project

Reply:
I thought verify was needed to ensure plugins like checkstyle and the javadoc linting kick in. If they are running as part of package then it can be reverted

Reply:
fixed in new commit